### PR TITLE
Kube scheduler v1beta2

### PR DESCRIPTION
**Pull request by tengqm**
This PR updates the module dependency in preparation for 1.22 release. Config for kube-scheduler v1beta1 is now replaced by v1beta2. New version of kubeadm-config v1beta3 is introduced.
Will follow up with other packages.

### CODE REVIEW

File: genref/go.mod [hunk 1]
```diff
@@ -3,31 +3,35 @@ module github.com/kubernetes-sigs/reference-docs/genref
...
+	github.com/go-openapi/spec v0.0.0-20160808142527-6aced65f8501 // indirect
+	github.com/google/go-cmp v0.5.5 // indirect
+	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/kr/text v0.2.0 // indirect
-	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
+	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e
 	github.com/pkg/errors v0.9.1
-	github.com/tengqm/kubeconfig v0.0.0-20210703133455-61479ab64f9d
-	github.com/yuin/goldmark v1.2.1
+	github.com/tengqm/kubeconfig v0.0.0-20210707141017-7849c7b17290 // indirect
```
Comment (sftim) [inline genref/go.mod:13]: I'm not sure what this new indirect dependency (at version 0.0.0) is doing / why it's included. It stands out as different from the others.

File: genref/config.yaml [hunk 2]
```diff
@@ -25,6 +25,9 @@ markdownDisabled: false
 stripPrefix:
   - k8s.io/api/
   - k8s.io/apimachinery/pkg/apis/
+  - github.com/tengqm/kubeconfig/config/kubeadm/v1beta2.
```
Comment (kbhawkey) [inline genref/config.yaml:28]: Hi @tengqm . Are these strings correct. I'm asking about the period at the end.

File: genref/go.mod (hunk 1 above)
Comment (kbhawkey) [reply genref/go.mod:13]: I guess this is okay.

**Comment by kbhawkey** [comment]
The changes look okay. Thanks @tengqm .
Holding, but feel free to remove the hold.
/lgtm
/hold

File: genref/config.yaml (hunk 2 above)
Comment (tengqm) [reply genref/config.yaml:28]: The period was intentional. Thanks for asking.

File: genref/go.mod (hunk 1 above)
Comment (tengqm) [reply genref/go.mod:13]: the `go mod` command sometimes adds the `// indirect` comment, sometimes it doesn't.

**Pull request merged**